### DISCUSSION TITLE
Post man Flow does not iterate over the list of elements, selecting of a variable is disabled.

### DISCUSSION THREAD

**Discussion by sachincoolkarni** [postmanlabs/postman-flows#158] [Defect]
![image](https://user-images.githubusercontent.com/8046649/145792494-fb9963f0-882b-4e77-b03e-da4c52250c91.png)

**Comment by samay-v** [comment]
Hi @sachincoolkarni, 
Could you show us a preview of the response body that you're trying to iterate over? 
Flows can iterate over elements of a `List`. Seems like your input does not have a `List`, but only `Records`. You can understand more about Abstract data types [here](https://github.com/postmanlabs/postman-flows/blob/main/docs/data-types.md#abstract-types)

Could you expand on the variables being disabled issue a bit?
